requirements.txt

```diff
@@ -1 +0,0 @@
-pennsieve2
```

Comment:
If this file is empty, why keep it?

Reply:
I don't see requirements.txt file. 
Isn't it getting removed with this pull request?